### DISCUSSION TITLE
Program

### DISCUSSION THREAD

**Discussion by kjappelbaum** [MADICES/MADICES-2022#20] [Ideas]
We need to come up with a (tentative) program. We have three days and want to cater to all time zones (US, Europe, Asia). 

Hence, we could have three sessions per day (UTC): 

- 9.00 -- 11.30 
- 14.00 -- 16.30
- 19.00 -- 21.30 

On the first day, the first session should start with a general pre-recorded intro that we make available on YouTube before the workshop. 

We should probably introduce every session with 1--2 min by the moderator for the specific session about the schedule. 

Within each session, we need to figure out: 
- What is the ratio presentation/breakout room 

Between sessions: 
- How do we manage the breakout rooms when new participants might join/others will leave: 
   - Does the moderator show the summary video from the previous session? 
   - On GitHub: Do we use one Markdown file per breakout room or one folder per room and then one file per session (I think the former is better)

**Comment by ml-evs** [comment]
Session times look good to me.

I think the plenary/keynote/panel will be hardest to schedule. We might even consider scrapping the Day 1 session 1 depending on participation (and end a bit earlier for a social session on Gather - will need to check with Aude, hopefully we could have it open without her needing to stay around)? Here is what I was imagining:

Day 1:

14:00 - 14:45 - Screening of introductory talks by organizers
14:45 - 15:30 - Keynote + questions
15:30 - 16:00 - Talks from 2 initiatives (preferably those that involve panel discussion members)
16:00 - 16:30 - Panel discussion
16:30 - 17:30 - Gather open for socialising

18:00 - 19:00 - Re-screening of intro talks
19:00 - 20:30 - Breakout 1
20:30 - 21:00 - Summing up Breakout 1 (which may decide future topics more than others)
21:00 - 21:30 - Gather open for socialising

Day 2: 

09:00 - 10:00 - Talks from 3-4 initiatives
10:00 - 11:30 - Breakout 2

14:00 - 14:30 - Summing up from Breakouts 1 & 2
14:30 - 15:00 - Talk from 2 initiatives
15:00 - 16:30 - Breakout 3

19:00 - 19:30 - Talk from 2 initiatives
19:30 - 21:00 - Breakout 4
21:00 - 21:30 - Summing up Breakouts 3 & 4

Day 3:
09:00 - 09:30 - Talk from 2 initiatives
09:30 - 11:00 - Breakout 5
11:00 - 11:30 - Summing up Breakouts 3, 4 & 5

14:00 - 15:30 - "Final" breakout
15:30 - 16:30 - "Final" summary for plans going forward

19:00 - 21:30 - Mixed breakouts, discussions and socialising, perhaps organizing team sits down and tries to pull everything together


This gives us 10 short talk slots and 6 1.5 hour breakout sessions to fill. We can adjust that ratio as we please.

**Alternatively:**

- move the panel to day 2 after more initiatives have spoken.
- Fill the first session on day 1 with initiative talks, but accept that they will be less motivated by the intro / replace the session with some Gather socialising for an hour for EU/Asia-based participants

**Reply by sphuber** [reply]
I am bit unclear on how the three sessions for different timezones are going to work. It looks like you are proposing that we are not "duplicating" any content, but all sessions will be unique. This will mean that many people will miss a lot of discussion if they only join one timezone session. I wonder if this won't give problems in continuity if the discussions are not fully independent. I guess that is what the "sum up" sessions are supposed to cover. Maybe there is no better solution indeed, but just wondering if you had concrete ideas about how this would pan out.

Will we have speakers and or participants from all three timezones, you reckon? I think USA and EU are pretty likely, but do we expect any (significant) participation from the Asia timezone?

**Reply by ml-evs** [reply]
> I am bit unclear on how the three sessions for different timezones are going to work. It looks like you are proposing that we are not "duplicating" any content, but all sessions will be unique. This will mean that many people will miss a lot of discussion if they only join one timezone session. I wonder if this won't give problems in continuity if the discussions are not fully independent. I guess that is what the "sum up" sessions are supposed to cover. Maybe there is no better solution indeed, but just wondering if you had concrete ideas about how this would pan out.

I agree it is tricky, and it really relies on some committed core of people to attend all 3 sessions (as well as the organizers) which is unlikely! Making the summaries as good as we can and easily available will be key, and we will have to make an effort to modularise the discussions within a given topic across the sessions to limit repetition. We currently have 7 potential topics A-G (I think we could come up with 10), some of which may only require 1-2 sessions, so its possible that you can get the full value from just attending 2 commensurate sessions on a given day. 

We could even make it so the same topics are discussed at the same times each day, if the attendee interest can also be nicely partitioned by timezone (e.g., I think all of the ELN developers we are reaching out to are currently EU based, and all of the facility people are NA based...).

> Will we have speakers and or participants from all three timezones, you reckon? I think USA and EU are pretty likely, but do we expect any (significant) participation from the Asia timezone?

NA/EU is inevitable (we have NA panel members already confirmed), but yes, we will probably have limited participation from attendees in Asia. I still think this is better than guaranteeing zero participation. The RDA interest group has significant participation from Asian timezones, as has OPTIMADE, so we may get a few registrations. I am sure there are EU people that would be happy to join for just a morning rather than the whole day, too.
